src/test/java/com/newestuser/imen/bgdate/BulgarianDateTest.java

```diff
@@ -25,4 +25,36 @@ private Date firstOfJanuary(int year) {
         calendar.set(Calendar.DAY_OF_MONTH, 1);
         return calendar.getTime();
     }
+
+    @Test
+    public void checkLeapYear() {
+        Date gregorianDate = secondOfJanuary(2020);
+        BulgarianDate bulgarianDate = new BulgarianDate(gregorianDate);
+        String formatedDate = bulgarianDate.format(bulgarianDate);
+        assertEquals(formatedDate, "11 Алем, 6788 Сомор");
+    }
+
+    private Date secondOfJanuary(int year) {
+        Calendar calendar = Calendar.getInstance();
+        calendar.set(Calendar.YEAR, year);
+        calendar.set(Calendar.MONTH, Calendar.JANUARY);
+        calendar.set(Calendar.DAY_OF_MONTH, 2);
+        return calendar.getTime();
+    }
+
+    @Test
+    public void checkLeapYearSecondTest() {
```

Comment:
this test name is misleading, when I read it I think that it should assert on a leap year, instead it asserts a non leap year `checkNonCenturialLeapYear` sounds like a better name